i2c-pull-up-resistors-intro.md

```diff
@@ -13,18 +13,18 @@ advisable to check the pull-up resistor configuration.
 Where can pull-up resistors be placed:
 
 1. On the development board you are connecting.
-2. On the microcontroller.
-3. Manually wired on the SDA and SCL lines between the microcontroller and development board, for example, by using a
+2. On the micro-controller.
```

Comment:
microcontroller is usually spelled as a single word without the '-'. Here default spell checking is wrong